recipes/docopt.cpp/all/patches/cmake-library-0.6.3.patch

```diff
@@ -35,8 +35,12 @@
  endif()
  
  if(USE_BOOST_REGEX)
-@@ -82,9 +75,6 @@ if(USE_BOOST_REGEX)
-     find_package(Boost 1.53 REQUIRED COMPONENTS regex)
+@@ -79,12 +72,9 @@ if(USE_BOOST_REGEX)
+ 	# This is needed on Linux, where linking a static library into docopt.so
+ 	# fails because boost static libs are not compiled with -fPIC
+ 	set(Boost_USE_STATIC_LIBS OFF)
+-    find_package(Boost 1.53 REQUIRED COMPONENTS regex)
++    find_package(boost 1.53 REQUIRED COMPONENTS regex)
```

Comment:
I am fairly certain this is should not be required... just adding the generator was no sufficient?

Reply:
Interesting! I thought lowercase was necessary but sure enough it works with `find_package(Boost ...)`. I just removed this change